frontend/src/views/Transfers/AddEditViewTransfer.jsx

```diff
@@ -210,14 +211,21 @@ export const AddEditViewTransfer = () => {
           }
         })
       }
+      alertRef.current.triggerAlert()
     },
     onError: (_error, _variables) => {
-      setAlertMessage(
-        transferId
-          ? t('transfer:actionMsgs.errorUpdateText')
-          : t('transfer:actionMsgs.errorCreateText')
-      )
+      const errorMsg = _error.response.data?.detail
```

Comment:
would also need to clear the confirmation modal. As it still persisting when we click on "Sign and send" button.

Reply:
I added the modal close, as well as a scroll to top so the user can see the error.